src/components/Shared/UserSearchField.tsx

```diff
@@ -97,7 +97,7 @@ class UserSearchField extends React.Component<IProps, IState> {
     return (
       <span>
         <span className={css.suggestionAvatar}>
-          <AccountImage accountAddress={suggestion.ethereumAccountAddress} />
+          <AccountImage accountAddress={suggestion.ethereumAccountAddress} profile={suggestion} />
```

Comment:
just a question: where do I see the effect of this? (I see no image in the contributionreward selection of beneficiary)